docs/src/main/sphinx/object-storage/legacy-azure.md

```diff
@@ -192,3 +192,52 @@ CREATE TABLE hive.abfs_export.orders_abfs
 WITH (format = 'ORC')
 AS SELECT * FROM tpch.sf1.orders;
 ```
+
+(fs-legacy-azure-migration)=
+## Migration to native Azure implementation
+
+Trino includes a [native Azure
+implementation](/object-storage/file-system-azure) that does not rely on the
+legacy Hadoop implementation. Trino strongly advises upgrading existing
```

Comment:
legacy Hadoop implementation .. is also wrong.. maybe "legacy support, that is facilitates libraries from the Hadoop, Hive, and HDFS ecosystem" or something like that